.github/workflows/spell_check.yml

```diff
@@ -17,3 +17,14 @@ jobs:
         uses: actions/checkout@v4
       - name: Check spelling
         uses: crate-ci/typos@master
+      - name: check macros
+        run: |
+          for file in $(git diff --name-only --diff-filter=A); do
+            ./scripts/check_macros.scp "$file" &>> indent_script_output.txt
```

Comment:
"indent_script_output.txt"
should be renamed to smth like "check_macros_output.txt"